mne/beamformer/_lcmv.py

```diff
@@ -352,7 +363,7 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.01, label=None,
 
 
 @verbose
-def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.01, label=None,
+def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.1, label=None,
```

Comment:
In Fieldtrip the regularization parameter is cfg.dics.lambda = '5%'.
In a tutorial, they suggest to change it to 0 or to '10%' to see the change in spatial resolution:
http://www.fieldtriptoolbox.org/tutorial/beamformer